source/beamline/VESPERS/VESPERSAddOnsCoordinator.cpp

```diff
@@ -1836,3 +1854,24 @@ void VESPERSAddOnsCoordinator::restoreAddOnsWireVAndNStatus()
 	if(!addOnsWireNStatusControl_->withinTolerance(finalStatus))
 		addOnsWireNStatusControl_->move(finalStatus);
 }
+
+void VESPERSAddOnsCoordinator::onMonoStatusChanged(bool isMoving)
+{
+	if (isMoving){
+
+		monoWaitTimer_->start(30000);
+		connect(monoFeedbackControl_, SIGNAL(valueChanged(double)), monoWaitTimer_, SLOT(start()));
```

Comment:
Nope.  While the move is active, we need to restart the timer every time we get a value changed signal.  Otherwise, we could conceivably stop a valid move (if a move took longer than 30 seconds).  Since we know that the move is working correctly if the value changed signal is being emitted, we can just have the timer restart it's timer (since start() is used as a restart in the Qt API).